packages/ra-ui-materialui/src/list/Pagination.js

```diff
@@ -16,6 +16,8 @@ const Pagination = ({
     total,
     setPage,
     setPerPage,
+    ActionsComponent,
```

Comment:
not sure about this name. I think in other places (or in material-ui), component props start with lowercase and don't include the term "component"

Reply:
You're right: list, create, show etc, are component (and not elements) yet their props are lowercase and without component at the end.
I submitted the change